src/ErrorPage/ErrorPage.js

```diff
@@ -6,10 +6,63 @@
 /* @flow */
 
 import React from 'react';
+import glamorous from 'glamorous';
 
 import history from '../history';
 import Link from '../Link';
-import s from './ErrorPage.css';
+
+const color = '#607d8b';
+
+const Container = glamorous.div({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  display: 'flex',
+  width: '100%',
+  height: '100%',
+  textAlign: 'center',
+  justifyContent: 'center',
+  alignItems: 'center',
+  '@media only screen and (max-width: 280px)': { width: '95%' },
+});
+
+const Main = glamorous.main({
+  paddingBottom: 80,
+  '@media screen and (max-width: 1024px)': { padding: '0 16px' },
+});
+
+const ErrorCode = glamorous.h1({
+  margin: 0,
+  fontSize: '15em',
+  fontWeight: '300',
+  lineHeight: '1em',
+  color,
+  letterSpacing: '0.02em',
+  '@media screen and (max-width: 1024px)': { fontSize: '10em' },
+});
+
+const Title = glamorous.p({
+  paddingBottom: '0.5em',
+  fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif',
+  fontSize: '2em',
+  fontWeight: 400,
+  lineHeight: '1em',
+  color,
+  letterSpacing: '0.02em',
+  '@media only screen and (max-width: 280px)': {
+    margin: '0 0 0.3em',
+    fontSize: '1.5em',
+  },
+  '@media screen and (max-width: 1024px)': { fontSize: '1.5em' },
```

Comment:
Might be handy to store the media query in a named variable so it can be shared and the code is easier to understand :)

Reply:
Yep, thanks.

Reply:
@kentcdodds so far it fills a little uncomfortable to write styles as JS objects, the need in quoting stuff is killing me :)

Reply:
Take a look at [this](https://atom.io/packages/css-in-js). As well as other projects referenced in [the glamorous repo](https://github.com/paypal/glamorous) :smile: